commands/utils/utils.go

```diff
@@ -64,3 +67,23 @@ func Md5Hash(values ...string) (string, error) {
 	}
 	return hex.EncodeToString(hash.Sum(nil)), nil
 }
+
+func UploadScanToGitProvider(scanResults []services.ScanResponse, params *FrogbotParams, client vcsclient.VcsClient) error {
+	// don't do anything if scanResults is empty
+	if xrayutils.IsEmptyScanResponse(scanResults) {
+		return nil
+	}
+	includeVulnerabilities := params.Project == "" && params.Watches == ""
+	scan, err := xrayutils.GenerateSarifFileFromScan(scanResults, includeVulnerabilities, false)
+	if err != nil {
+		return err
+	}
+	_, err = client.UploadScanningAnalysis(context.Background(), params.RepoOwner, params.Repo, params.BaseBranch, scan)
```

Comment:
IMO Scanning Analysis does not describe exatly what it does.
My suggestion - Code Scanning
```suggestion
	_, err = client.UploadCodeScanning(context.Background(), params.RepoOwner, params.Repo, params.BaseBranch, scan)
```

Reply:
done